src/Symfony/Bundle/FrameworkBundle/Tests/Functional/Bundle/ModernBundle/ModernBundle.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\ModernBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class ModernBundle extends Bundle
```

Comment:
shouldnt this technically live in the `src` folder as well? which raises another bigger question; do bundles have a kernel-like `$projectDir`?

Reply:
You're right, it should be in `src`

Reply:
`$bundle->getPath()` is more like `$kernel->getRootDir()`, it's the path where the bundle class/file is placed.

We may need a `$bundle->getBundleDir()` similar to `$kernel->getProjectDir()` assuming the bundle file is in `src`:
```php
public function getBundleDir()
{
    return \dirname(parent::getPath());
}
```
and use `$bundle->getBundleDir().'/public'` to introduce the new conventions, wdyt?

Reply:
in that case we should deprecate `getPath()` IMHO much like we did with `getRootDir()`

or we actually need it, in that case `getRootDir()` would be in fact the better name